src/components/fields/textfield/types.ts

```diff
@@ -0,0 +1,11 @@
+import { FormInputProps } from "@lifesg/react-design-system/form/types";
+import { IFrontendEngineBaseFieldJsonSchema, TFrontendEngineBaseFieldJsonSchemaKeys } from "../../frontend-engine";
+import { IValidationRule } from "../../frontend-engine/validation-schema/types";
+
+export interface ITextfieldSchema
+	extends IFrontendEngineBaseFieldJsonSchema<"TEXT" | "NUMBER" | "EMAIL">,
+		Omit<FormInputProps, TFrontendEngineBaseFieldJsonSchemaKeys> {
+	id: string;
```

Comment:
oops I think I missed this out, can remove `id`, `title`, `validation` here because they are already defined in `IFrontendEngineBaseFieldJsonSchema`